pyfar/plot/_interaction.py

```diff
@@ -58,18 +58,22 @@ class Cycle(object):
     """ Cycle class implementation inspired by itertools.cycle. Supports
     circular iterations into two directions.
     """
-    def __init__(self, n_channels, index=0):
+    def __init__(self, cshape, index=0):
         """
         Parameters
         ----------
-        n_channels : int
-            number of channels in the signal
+        chsape : tupel
```

Comment:
```suggestion
        chsape : tupel, array_like, int
```

Reply:
Yes, changed as it works with all! We only pass tuples so far...